src/ptf/mask.py

```diff
@@ -1,8 +1,8 @@
 from __future__ import print_function
+import warnings
 from six import StringIO
 import sys
-from scapy.utils import hexdump
-from . import packet as scapy
+from . import packet
```

Comment:
Is this import works with Python 3?

Reply:
![image](https://user-images.githubusercontent.com/16683011/143233603-b9d7df62-41d4-4d57-81bb-564d1727c1a1.png)
I tested it on python 3.6.8